src/server/game/Handlers/QuestHandler.cpp

```diff
@@ -117,7 +117,7 @@ void WorldSession::HandleQuestgiverAcceptQuestOpcode(WorldPacket& recvData)
 #define CLOSE_GOSSIP_CLEAR_DIVIDER() \
```

Comment:
Rename this macro while you're at it too, please (`CLOSE_GOSSIP_CLEAR_SHARING_INFO`)